pkg/tools/gen/types.go

```diff
@@ -24,6 +24,89 @@ const (
 	typNumberMultiplier = "number_multiplier"
 )
 
+type KclSchema struct {
+	Imports []string
+	Schemas []Schema
+}
+
+type Schema struct {
+	Name        string
+	Description string
+	Properties  []Property
+}
+
+type Property struct {
+	Name         string
+	Description  string
+	Type         TypeInterface
+	Required     bool
+	HasDefault   bool
+	DefaultValue interface{}
+}
+
+type TypeInterface interface {
+	Format() string
+}
+
+type TypePrimitive string
+
+const (
+	Str   TypePrimitive = "str"
```

Comment:
dup definitions at types.go line 14-24

Reply:
You are right, I will remove these duplicate codes